cherrypy/_cpreqbody.py

```diff
@@ -589,18 +585,19 @@ class Part(Entity):
     module in Python's standard library.
     """
 
-    def __init__(self, fp, headers, boundary):
+    def __init__(self, fp, headers, boundary, charset):
+        self.attempt_charsets = [charset]
```

Comment:
What's the purpose of having a list of attempted charsets if there's always exactly one?